configs/secureauth.json

```diff
@@ -18,6 +18,23 @@
     "lvl5": "section h6",
     "text": "section p, section li"
   },
+  "custom_settings": {
+    "attributesForFaceting": ["version"],
```

Comment:
If the `docsearch:version` meta tag is present, it will override values from https://github.com/algolia/docsearch-configs/blob/62516ed7ba32405063b93de2b4063caac1a97965/configs/secureauth.json#L5-L7